src/states/Menu.js

```diff
@@ -1,20 +1,126 @@
-var Menu = function () {};
+/* globals window */
 
-Menu.prototype = {
+//In later changes I may want to create a list of buttons rather than store each individually
+//It may be nice to have something more dynamic
+var Menu = function () {
+  this.backDropFilter = null;
+  this.backDrop = null;
+  this.titleText = null;
+  this.startButton = null;
+  this.optionButton = null;
+  this.fullscreenButton = null;
+  this.muteButton = null;
+  this.seedButton = null;
+  this.returnButton = null;
 
-  preload: function() {
+};
 
+Menu.prototype = {
+  preload: function() {
+    window.game.load.script('sunset', 'assets/shaders/MenuShader.js');
   },
 
   create: function() {
-
+    window.game.stage.backgroundColor = '#FFFFFF';
+    window.game.scale.fullScreenScaleMode = Phaser.ScaleManager.NO_SCALE;
+    //load Filter
+    //this.initMenuBackground();
+    //Apply filter to image in background
```

Comment:
Personally I think using 
`/* Apply filter to image in background */`
looks much nicer for "section" comments, I would use `//` for writing comments beside a specific line. That's my personal preference cause I think it looks alot nicer, up to you if you want to change that though.